src/AspNetCoreModuleV2/AspNetCore/applicationmanager.cpp

```diff
@@ -189,12 +189,6 @@ APPLICATION_MANAGER::RecycleApplicationFromManager(
 
         // Removed the applications which are impacted by the configurtion change
         m_pApplicationInfoHash->DeleteIf(FindConfigChangedApplication, (PVOID)&context);
-
-        if (m_pApplicationInfoHash->Count() == 0 && m_hostingModel == HOSTING_OUT_PROCESS)
-        {
-            // reuse current process
-            m_hostingModel = HOSTING_UNKNOWN;
```

Comment:
Why remove this check? What happens when we run an out of process app, shut it down, then start an inprocess app?

Reply:
m_hostingModel  value never used. The logic you are talking about is in HandlerResolver